script/process-file-manifest.js

```diff
@@ -0,0 +1,50 @@
+function processFileManifest() {
```

Comment:
Is there a better name for this? I think it's confusing given that we have our own manifest files, in addition to the Webpack manifest. Maybe `addAssetHashes`?